docs/Getting-Started/Private-Network-Quickstart.md

```diff
@@ -227,62 +244,63 @@ The result of this call should be something like :
 ```
 
 !!!info
-    0x79f905c6fd34e80000 = 2250000000000000000000 Wei (2250 Ether)
-    
-    you can use a unit [converter](https://etherconverter.online/) to go from wei to ether.
+    0x79f905c6fd34e80000 = 2250000000000000000000 Wei (2250 Ether).
+    Use a [unit converter](https://etherconverter.online/) to translate values from Wei to Ether.
```

Comment:
"you can" seemed appropriate as there's many other ways to do that including manual conversion. Perhaps use a tip there.
```suggestion
!!!tip
    Use a [unit converter](https://etherconverter.online/) to easily translate values from Wei to Ether.
```

Reply:
Yes--I think it's okay to use "you can" as this is a helpful option.